data-model.d.ts

```diff
@@ -288,3 +295,8 @@ export interface DataFactory<OutQuad extends BaseQuad = Quad, InQuad extends Bas
      */
     quad(subject: InQuad['subject'], predicate: InQuad['predicate'], object: InQuad['object'], graph?: InQuad['graph']): OutQuad;
 }
+
+export interface DirectionalLanguage {
+    language: string;
+    direction?: 'ltr' | 'rtl' | '';
```

Comment:
In what cases would `''` be used as input over `undefined`

Reply:
Because `direction` can be a falsy value, as discussed in the PR here: https://github.com/rdfjs/data-model-spec/pull/175

That actually reminds me, we should also allow `null` here...